report-viewer/src/components/DistributionDiagram.vue

```diff
@@ -51,7 +51,7 @@ const chartData = ref({
   datasets: [
     {
       ...dataSetStyle.value,
-      data: props.distribution.distribution
+      data: props.distribution.getTenthPercentileFormattedValues()
```

Comment:
You could use `perMille` here (if that's what you mean to express) https://en.wikipedia.org/wiki/Per_mille

Reply:
No thats not what I mean, but now that you mention it I can see where the confusion comes from and we should probably search for a better name.
The function does is return the distribution in the "old" way, so summed up to every thenth percentile (so 0%-10%, 10%-20%, ...). 
Do you think it would be clear enough if the funtion was named `getTenthPercentileSumDistribution()`?

Reply:
I read "tenth" as `1/10`, `TenPercentile-something` would be better for me.

However, the name seems complicated to grasp and very specific. I might write a somewhat more generic `splitInEqualBuckets(N)` function, and then stick `10` in as an argument.

If you want a dedicated function for it, something like `splitIntoTenBuckets` or so?